src/models/event.js

```diff
@@ -0,0 +1,37 @@
+import { simpleGet, simplePost } from '../shared/RxAjax';
+import Analyses from './analyses';
+import MusitObject from './object';
+import { parseISODate, DATE_FORMAT_DISPLAY } from '../shared/util';
+import MusitActor from './actor';
+import uniq from 'lodash/uniq';
+import { I18n } from 'react-i18nify';
+import concat from 'lodash/concat';
+import { Observable } from 'rxjs';
+
+export default class Event {}
+
+Event.concatAnalysesWithMoves = (ajaxGet = simpleGet, ajaxPost = simplePost) => (val) =>
```

Comment:
Should this have another name:  analysesAndMovesForObjects, or something like that?